docs/toc.json

```diff
@@ -71,6 +71,9 @@
       "title": "Region",
       "type": "compute/region"
     }, {
+      "title": "Service",
+      "type": "compute/service"
+    }, {
```

Comment:
@callmehiphop it's in here! Did I do something wrong? Also, is `implemented` still a thing?